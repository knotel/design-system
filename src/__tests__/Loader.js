import React from 'react'
import renderer from 'react-test-renderer'
import { Loader } from '..'

describe('Loader', () => {
  test('renders', () => {
    const json = renderer.create(<Loader />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
