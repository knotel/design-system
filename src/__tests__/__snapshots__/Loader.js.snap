// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders 1`] = `
<div
  className="rai-activity-indicator undefined"
  style={
    Object {
      "display": "inline-block",
      "fontSize": "16px",
      "lineHeight": "0",
    }
  }
>
  <div
    className="rai-sentry"
    style={
      Object {
        "animationDuration": "1.142857142857143s",
        "fontSize": 35,
      }
    }
  >
    <div
      className="rai-wave-container"
    >
      <div
        className="rai-wave"
        style={
          Object {
            "borderColor": "#E74C3C",
          }
        }
      />
    </div>
    <div
      className="rai-wave-container"
    >
      <div
        className="rai-wave"
        style={
          Object {
            "animationDelay": "-0.5714285714285715s",
            "borderColor": "#E74C3C",
          }
        }
      />
    </div>
  </div>
</div>
`;
